### PR TITLE
docs: Add new feature request template

### PR DESCRIPTION
**Pull request by s3cube**
Introduce a new feature request template to stream-line requests

### CODE REVIEW

**Review by omkhegde — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new-feature-template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+labels: feature-request
+assignees: ''
+
+---
+
+## Feature Description
+A clear and concise description of the feature you're proposing.
+
+## Problem Statement
+Describe the problem this feature would solve
```
[the remark this answers is not in the document]
Comment (s3cube) [reply .github/ISSUE_TEMPLATE/new-feature-template.md:14]: That should be covered by the next point `Alternatives Considered`?

[the remark this answers is not in the document]
Comment (kg-aws) [reply .github/ISSUE_TEMPLATE/new-feature-template.md:14]: In my opinion, the `Alternatives Considered` has a border meaning. As the description for `Alternatives Considered` says `Describe any alternative solutions or features you've considered` but doesn't exactly prompt customer if they are taking any intermediary steps while they request a nice to have feature.

We could also update the description of `Alternatives Considered` to add the `steps taken to circumvent if applicable.`

[the remark this answers is not in the document]
Comment (s3cube) [reply .github/ISSUE_TEMPLATE/new-feature-template.md:14]: Makes sense, updating this

**Review by kg-aws — APPROVED** [review]
LGTM!

**Pull request merged**